sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -490,7 +496,7 @@ func (m *networkFlowManager) enrichProcessListening(ep *containerEndpoint, statu
 
 	// Multiple endpoints from a collector can result in a single enriched endpoint,
 	// hence update the timestamp only if we have a more recent endpoint than the one we have already enriched.
-	if oldTS, found := processesListening[indicator]; !found || oldTS < status.lastSeen {
+	if oldTS, found := processesListening[indicator]; !found || oldTS < status.lastSeen || oldTS == timestamp.InfiniteFuture {
```

Comment:
It seems likely that this change is not needed, as if it were a bug it would be negatively impacting other parts of the code that have been in production for a long time. Need to check this. Similar comment for other similar changes.

Reply:
Removed